.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Description of change
+
+_Please provide description of change here_.
+
+### Pull Request check-list
+
+_Please make sure to review and check all of these items_:
+
+- [ ] Does `make test` (or `vcbuild test nosign`) pass with this change?
```

Comment:
There are some changes that do not require testing atm. For example, the docs are not tested afaik (or are they now)?